addons/l10n_lv/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,659 @@
+        <field name="price_include" eval="0"/>
+        <field name="amount">0</field>
+        <field name="amount_type">percent</field>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="tax_group_vat_exempt"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_report_expression_ids': [ref('l10n_lv_tax_report_482_tag')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_report_expression_ids': [ref('l10n_lv_tax_report_482_tag')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+    </record>
+
+    <!--  VAT, VAT report PVN2 services-->
+    <record id="account_tax_template_sales_VAT_2P_tag" model="account.tax.template">
+        <field name="chart_template_id" ref="account_chart_template_latvia"/>
+        <field name="sequence">302</field>
+        <field name="name">VAT2 Services</field>
+        <field name="description">W/O VAT (EU)</field>
+        <field name="price_include" eval="0"/>
+        <field name="amount">0</field>
+        <field name="amount_type">percent</field>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="tax_group_vat_exempt"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_report_expression_ids': [ref('l10n_lv_tax_report_482_tag')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'minus_report_expression_ids': [ref('l10n_lv_tax_report_482_tag')],
+                }),
+
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax',
+                }),
+            ]"/>
+    </record>
+
+    <!--  VAT, VAT report PVN2 goods-->
+    <record id="account_tax_template_sales_VAT_2G_tag" model="account.tax.template">
+        <field name="chart_template_id" ref="account_chart_template_latvia"/>
+        <field name="sequence">202</field>
```

Comment:
We need the sequence numbers to keep increasing in order for the taxes to be displayed in the same order as in the file :smile:.